python/front_end/znetio.py

```diff
@@ -194,7 +205,7 @@ def consolidate_opts(source_opts, dest_opts, params=None, layers=None, is_seed=F
     for group_type in range(len(dest_opts)):
         for opt_dict in dest_opts[group_type]:
 
-            if is_seed and opt_dict.has_key('seeding') and (opt_dict['seeding'] == "0"):
+            if opt_dict.has_key('seeding') and (opt_dict['seeding'] == "0"):
```

Comment:
@jingpengwu I added it. That's an option used in nodes/edges spec, and it indicates that we don't wanna seed the nodes/edges with `seeding=0`.